components/Repositories/LotteryFactorChart.tsx

```diff
@@ -129,6 +134,26 @@ export default function LotteryFactorChart({
         </header>
       </section>
 
+      {yoloBannerOnClick && (
+        <button
+          onClick={yoloBannerOnClick}
+          className="flex items-center justify-between w-full text-slate-500 shadow-sm !border !border-slate-300 p-1 gap-2 text-sm rounded-full"
+        >
+          <div className="flex gap-2 items-center">
+            <div className="flex items-center font-medium gap-1 px-2 py-0.5 rounded-2xl bg-light-red-4 text-light-red-11">
+              <FaRegHandPeace />
+              YOLO Coders
+            </div>
+            <p className="hidden xl:block">Pushing commits directly to main</p>
+          </div>
+
+          <div className="flex gap-2 items-center ml-auto mr-3">
+            <p>See more</p>
```

Comment:
The see more is kind of implied by the arrow in the button and doesn't appear to be in the designs.


It also looks jam packed at the start of the XL breakpoint. Is the See more necessary?

![CleanShot 2024-07-02 at 17 44 12](https://github.com/open-sauced/app/assets/833231/d4502f3b-ae15-4d2f-96f5-f2bb87c6a7c4)


![CleanShot 2024-07-02 at 17 39 35](https://github.com/open-sauced/app/assets/833231/8b1a4f66-7434-448e-ad09-e539bdff02ad)

![CleanShot 2024-07-02 at 17 42 12](https://github.com/open-sauced/app/assets/833231/3ca2498d-907d-4bbb-bf5c-7e72d5448766)

Reply:
This would be a @isabensusan question since this was from her review:
> can we add a "Show more" text next to the arrow to fill the button more and make it feel less empty?

I've gotten rid of the "Pushing commits" description when it comes to smaller screens to help with the breakpoints, but I can do the same for the "See More" to only show on xl screens.

Reply:
e.g.

![CleanShot 2024-07-02 at 17 58 06](https://github.com/open-sauced/app/assets/833231/68d0509f-8cc6-45be-9a8d-2c1025072426)

![CleanShot 2024-07-02 at 17 58 26](https://github.com/open-sauced/app/assets/833231/c928b02f-9fea-4bc1-a41c-f3046839a8d6)

Reply:
I don't love the centered look, but I feel there's some breakpoints we can establish to make this look nice. 
The priority of info we want to show on the component is basically:
1. YOLO Coders badge
2. "Pushing commits directly to main"
3. Arrow
4. See more (this is more of an add on for when the component is too wide)

Here's an example of how we can play around with what we show when:

Anything below the xs breakpoint (<425px)
- we can put the "directly" portion of the text in a span and make it hidden
<img width="430" alt="image" src="https://github.com/open-sauced/app/assets/27322879/82170c5e-d5c5-4baa-a901-b179641f75a2">

xs breakpoint (425px > 640px)
- bring back the "directly" span and the arrow
<img width="655" alt="image" src="https://github.com/open-sauced/app/assets/27322879/095a4dff-5066-4567-aed3-67c1f304ac13">


sm to lg (640px > 1024px)
- bring in the "See More"
<img width="1020" alt="image" src="https://github.com/open-sauced/app/assets/27322879/c02b24df-edaa-4c97-8df9-1450ea318c7d">


lg to xl (1024px > 1280px)
<img width="949" alt="image" src="https://github.com/open-sauced/app/assets/27322879/72474ebc-03e7-4265-8f43-bf4d5dc384f3">

xl to 1800px (this xxl breakpoint doesn't exist, but it's where it breaks)
<img width="1612" alt="image" src="https://github.com/open-sauced/app/assets/27322879/67452d6f-9c3a-4878-9069-4c4f3738f98e">

Anything > 1800px 
- Add the "See More"
<img width="2524" alt="image" src="https://github.com/open-sauced/app/assets/27322879/ec824a1f-0290-4674-8438-d6943440f37c">


I know it seems very complex, but it's just a few tailwind classes to make this look really polished